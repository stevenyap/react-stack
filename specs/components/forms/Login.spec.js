import React from 'react'

import LoginForm from 'components/forms/Login'

describe('LoginForm', () => {
  it('renders correctly', () => {
    const tree = renderJSON(<LoginForm />)
    expect(tree).toMatchSnapshot()

    const renderedText = toText(tree)
    expect(renderedText).toContain('Email')
    expect(renderedText).toContain('Password')
    expect(renderedText).toContain('Submit')
  })
})
