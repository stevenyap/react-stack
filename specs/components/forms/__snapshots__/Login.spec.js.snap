// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm renders correctly 1`] = `
<form
  onSubmit={undefined}
>
  <div
    className="form-group has-feedback"
  >
    <label
      className="control-label"
      htmlFor={undefined}
    >
      Email:
    </label>
    <input
      className="form-control"
      id={undefined}
      onBlur={undefined}
      onChange={[Function]}
      type="text"
      value={undefined}
    />
  </div>
  <div
    className="form-group has-feedback"
  >
    <label
      className="control-label"
      htmlFor={undefined}
    >
      Password:
    </label>
    <input
      className="form-control"
      id={undefined}
      onBlur={undefined}
      onChange={[Function]}
      type="password"
      value={undefined}
    />
  </div>
  <button
    className="btn btn-success"
    disabled={false}
    onClick={undefined}
    type="submit"
  >
    Submit
  </button>
</form>
`;
